src/System.Private.CoreLib/shared/System/Diagnostics/Debug.cs

```diff
@@ -215,7 +215,7 @@ public static void Write(string message, string category)
             }
             else
             {
-                Write(category + ":" + message);
+                Write(category + ": " + message);
```

Comment:
Trace.Write(string message, string category) applies this extra space but Debug.Write(string message, string category) didn't.
To make them consistent I updated Debug to behave the same as Trace.

TODO: check what desktop does and make behavior same as what Desktop does instead, for both Trace and Debug

Reply:
Just confirmed Desktop also has the extra whitespace after `:`.

So this fix will make `Write(sting message, string category)` behavior the same for Debug and Trace on both .NET Core and Desktop.